components/bookingFormPopup/bookingFormPopup.module.css

```diff
@@ -0,0 +1,39 @@
+@keyframes formEnterOverlayAnimation {
```

Comment:
> is this type of animation used elsewhere?

No - looking at the old site, this is the only use case.

I tried moving the animations to the config but couldn't get it to work.
Because of the way these classes are used (not in a normal classname attribute), Tailwind doesn't recognise the usage and wont include the class in the build.

```tsx
<Modal
        open={isVisible}
        onClose={showBookingForm}
        showCloseIcon={false}
        classNames={{
          modalAnimationIn: "animate-formEnterModalAnimation", {/* tailwind doesnt see this */}
          modalAnimationOut: "animate-formLeaveModalAnimation", {/* tailwind doesnt see this */}
          modal: "w-[600px] bg-black/0 max-w-none"
        }}
        animationDuration={700}
      >
```